### DISCUSSION TITLE
Not showing photos

### DISCUSSION THREAD

**Discussion by mimjey** [community/community#28102] [Actions]
Hello
I have a problem
After I put the website I had designed on the host, I realized that the browser does not display the photos, even though I have uploaded the photos along with the html and css files.
What is the problem?

pictures format : svg  & png

**Comment by airtower-luna** [comment]
What kind of a "host"? If you mean GitHub pages, please link the relevant repository so people can take a look.

**Comment by jfullstackdev** [comment]
these are the errors

![image](https://user-images.githubusercontent.com/47092464/182660999-e5be17e1-a801-4787-8639-dde15140e1d0.png)

this is the correct URL 

<https://mimjey.github.io/mimjey/Images/persian.png>

![image](https://user-images.githubusercontent.com/47092464/182661505-0d1c4895-e4b7-4e23-b765-de00e8d98ae3.png)

but on your source, it's not like that, if you are still confused, use the absolute URL, one example is my URL above

![image](https://user-images.githubusercontent.com/47092464/182661361-1269c254-fb8e-4c05-b595-ed1d1fcba980.png)

**Comment by airtower-luna** [comment]
That looks like a case of case confusion. The directory in the repository is named `Images/` (with a capital `I`), but the `<img>` tags use `images/` (lower case). They need to be exactly the same to work. Either rename the directory to `images/`, or change the links to match the actual folder.

@jdevstatic, thanks for pointing out the repository!
